### PR TITLE
improve(hive):add health check

### PR DESCRIPTION
**Pull request by liebstein**

### CODE REVIEW

File: docker/hive/health-check.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ev
+
+HIVE_SERVER_PORT=${HIVE_SERVER_PORT:=10000}
+
+echo Health check for Hive:
+/opt/hive/bin/beeline -u "jdbc:hive2://127.0.0.1:${HIVE_SERVER_PORT}/default;auth=noSasl" -n root -p pass -e "show tables;"
```
Comment (Irenez753) [inline docker/hive/health-check.sh:7]: If Hive is unresponsive, so an error will be raised. Are we gonna see it in logs?

File: docker/hive/Dockerfile [hunk 2]
```diff
@@ -86,6 +86,7 @@ RUN chmod +x /wait
 COPY init-hive.sh /
```
Comment (Doe-Ed) [inline docker/hive/Dockerfile:86]: I know this is semi-related but why do we copy init-hive if we don't use it in microservices/production/hive-2.tf

Comment (liebstein) [reply docker/hive/Dockerfile:86]: We use it in start-hive.sh

Comment (Doe-Ed) [reply docker/hive/Dockerfile:86]: Thanks

File: docker/hive/health-check.sh (hunk 1 above)
Comment (lyogev) [reply docker/hive/health-check.sh:7]: Allow setting the user/password from outside?

Comment (liebstein) [reply docker/hive/health-check.sh:7]: These are mock creds since we use noSasl

Comment (liebstein) [reply docker/hive/health-check.sh]: @lyogev Apparently NoSasl requires beeline user but not password. I changed the user to `health_check`

**Review by Irenez753 — APPROVED** [review]

**Pull request merged**